index.html

```diff
@@ -274,6 +274,229 @@ <h3>
         </ol>
       </section>
     </section>
+    <section>
+      <h2>
+        `protocol_handlers` member
+      </h2>
+      <p>
+        The [=manifest's=] <code><dfn data-dfn-for=
+        "manifest">protocol_handlers</dfn></code> member is an array of
+        <a>protocol handler description</a>s that allows a web application to
+        handle URL protocols.
+      </p>
+      <p>
+        On installation, a user agent SHOULD register protocol handlers with
+        the Operating System via interactions that are consistent with:
+      </p>
+      <ul>
+        <li>If there are multiple registered handlers for a protocol, the OS
+        allows the user to select which app should open it, and also allows the
+        user to set a default app.
+        </li>
+        <li>clicking on a registered protocol will launch the registered
+        application. If this is the web app, then execute the [=invoke a
+        protocol handler=] steps defined in [=HTML=], where the user agent
+        SHOULD navigate to [=url=] and the appropriate browsing context is set
+        to a new top level browsing context.
+        </li>
+      </ul>
+      <aside class="note">
+        Protocol handlers could, for instance, be used for web app
+        communication where one app directly invokes another and passes data
+        via custom protocol links.
+      </aside>
+      <section>
+        <h3>
+          Processing the `protocol_handlers` member
+        </h3>
+        <p>
+          To <dfn>process the `protocol_handlers` member</dfn>, given
+          [=object=] |json:JSON|, |manifest:ordered map|, and |start URL:URL|:
+        </p>
+        <ol>
+          <li>Let <var>processedProtocolHandlers</var> be a new [=list=] of
```

Comment:
Don't use <var> syntax, instead use |pipe| syntax for variables.

Check out https://respec.org/docs/